docs/getting-started/create-configuration.md

```diff
@@ -571,10 +571,10 @@ metadata:
     vpc: default
 spec:
   crossplane:
-    version: ">=v1.0.0-0"
+    version: ">=v1.2.0-0"
   dependsOn:
     - provider: crossplane/provider-aws
-      version: ">=v0.14.0"
+      version: "v0.18.1"
```

Comment:
Probably worth pinning this to v0.18.2 - we fixed a pretty serious bug in v0.18.1 since this PR was last updated. :)

Reply:
@negz I just updated the version to `v0.18.2`. Thanks

Reply:
Thank you! If you can sign your last commit I'll merge.

See https://github.com/crossplane/crossplane/pull/2309/checks?check_run_id=3567031276 for details.

Reply:
my bad, done.

Reply:
Please tell me if there's something else missing @negz

Reply:
I think we're all good now! Thanks for your persistence. :)